src/test/java/com/puppycrawl/tools/checkstyle/ant/CheckstyleAntTaskTest.java

```diff
@@ -946,8 +946,8 @@ public void testLoggedTime() throws IOException {
         final List<MessageLevelPair> loggedMessages = antTask.getLoggedMessages();
 
         assertLoggedTime(loggedMessages, testingTime, "Total execution");
-        assertLoggedTime(loggedMessages, testingTime, "To locate the files");
-        assertLoggedTime(loggedMessages, testingTime, "To process the files");
+        assertLoggedTime(loggedMessages, testingTime, "File discovery took");
```

Comment:
```suggestion
        assertLoggedTime(loggedMessages, testingTime, "File gathering took");
```
idc what about `details`, only about being more `concise`.

How you want this @romani  @nrmancuso ?

Reply:
follow up